src/zope/app/apidoc/codemodule/browser/introspector.rst

```diff
@@ -27,20 +27,22 @@ and the name of the object:
 Of course, the root folder does not have a name. As you can see the type links
 directly to the API documentation of the class.
 
-The next section lists all directly provided interfaces. The root folder
-directly provides the ``ISite`` and ``IRootFolder`` interface, so we should
-see those:
+The next section lists all directly provided interfaces. The root
+folder directly provides the :class:`zope.site.interfaces.ISite` and
+:class:`zope.site.interfaces.IRootFolder` interface, so we should see
+those::
```

Comment:
Whitespace in the block in rendered differently by Sphinx. With one colon, you get a separate section every time there's a blank line:
![one](https://user-images.githubusercontent.com/1256082/28122050-b3ab9d6c-66e2-11e7-8ebb-43966a60bf7f.png)

But with two colons, the entire block is treated as one example when rendered:
![two](https://user-images.githubusercontent.com/1256082/28122061-be1651ca-66e2-11e7-82d5-431fdac40b9b.png)

I think the two-colon version is less cluttered looking and also expresses the intent that this is meant to be read as one example better. The [sphinx docs](http://www.sphinx-doc.org/en/stable/rest.html#source-code) seem to suggest that `::` is the preferred spelling.

Plus, Emacs highlights the two types of blocks differently in the source 😄